tests/test_validate.py

```diff
@@ -511,7 +510,7 @@ def test_predicate_repr():
     assert (
         repr(validate.Predicate(method='foo', error='bar', zoo=1)) ==
         '<Predicate(method={!r}, kwargs={!r}, error={!r})>'
-        .format('foo', {str('zoo') if PY2 else 'zoo': 1}, 'bar')
+        .format('foo', {'zoo': 1}, 'bar')
```

Comment:
We'll probably want to switch to f-strings, but this is fine.

Reply:
Oh scratch that. I forget that f-strings aren't in 3.5, which we're supporting.

Reply:
3.5 EoL in 2020-09-13 so it probably will be still a while